### ISSUE TITLE
Question about fine tuning a V2 model

### ISSUE THREAD

**Issue by GUUser91** [Plachtaa/seed-vc#174]
I've noticed the V2 model config file doesn't have a max_len settings like the V1 model config file has.  Is there a recommendation for the highest duration for a single audio file, like say a 5-10 seconds audio file in a training set, or does it not matter for the V2 model training?

**Comment by Plachtaa** [comment]
0.2~45 seconds is recommended for training data
